Makefile

```diff
@@ -38,10 +38,10 @@ rebuild:
 #############################
 
 mysql-backup:
-	docker-compose run --rm --no-deps app root bash /docker/bin/backup.sh mysql
+	docker exec -it $$(docker-compose ps -q app) bash /docker/bin/backup.sh mysql
```

Comment:
No it's not. Inside a bash script that would be true, but if you want to execute a command inside a Makefile inline, you need to use double $. At least that's what I found out using Google and it only works that way. There might be some other escaping, but I don't know. That way it works.